src/Memstate.Docs.GettingStarted/QuickStart/QuickStartTests.cs

```diff
@@ -71,11 +73,61 @@ public async Task Simple_end_to_end_getting_started_using_default_settings_with_
             await engine.DisposeAsync();
         }
 
+        // rules for using Json serialisation
+        // ----------------------------------
+        // Commands must have public empty constructors
+        // properties must have { get; set; } or at minimum { get; private set; }
+
```

Comment:
No, not correct. the serializer will handle private/public fields, readonly fields, public and private properties.

Reply:
these comments are only for the Json Serializer. I found that if I create readonly fields then the Json test fails, so this may be the current state and we may want to add int the comments that we hope to change this? Please try this, change the `TransferPoints` dto for example and make the fields read-only, and the Json getting started test will fail. I'd be happy to be wrong here.

Reply:
Odd, I have a test that successfully clones https://github.com/DevrexLabs/memstate/blob/master/src/Memstate.Test/SerializationTests.cs#L82-L90